build.rs

```diff
@@ -84,6 +90,20 @@ fn main() {
     if linux_time_bits64 {
         set_cfg("linux_time_bits64");
     }
+    println!("cargo:rerun-if-env-changed=RUST_LIBC_UNSTABLE_GNU_FILE_OFFSET_BITS");
+    match env::var("RUST_LIBC_UNSTABLE_GNU_FILE_OFFSET_BITS") {
+        Ok(val) if val == "64" => {
+            if target_env == "gnu"
+                && target_os == "linux"
+                && target_ptr_width == "32"
+                && target_arch != "riscv32"
+                && target_arch != "x86_64"
+            {
+                set_cfg("gnu_file_offset_bits64");
+            }
+        }
+        _ => {}
```

Comment:
Could you add something like `Ok(val) if val != "32" => panic!("RUST_LIBC_UNSTABLE_GNU_FILE_OFFSET_BITS may only be set to '32' or '64'),`?

Reply:
I've added that check for both build.rs and libc-test/build.rs.